pkg/config/cluster.go

```diff
@@ -79,6 +80,10 @@ func (m *Cluster) DefaultAndValidate() error {
 		return errors.Wrap(err, "network configuration is invalid")
 	}
 
+	if err := m.Features.Validate(); err != nil {
+		return err
```

Comment:
Usually, we expanded those error to include what validation failed, e.g. `oidc validation failed`.

Reply:
✔️